exporters/webhook/plugins/pelorus_handler.py

```diff
@@ -71,16 +164,12 @@ class PelorusWebhookHandler(PelorusWebhookPlugin):
             "namespace": "mongo-persistent",
             "timestamp": 1557933657
         }
-
-    Attributes:
-        handshake_headers: (Headers): Headers that are received by the webhook.
-        request: (Request): The request object associated with the webhook.
     """
 
     user_agent_str = "Pelorus-Webhook/"
 
-    def __init__(self, handshake_headers: Headers, request: Request) -> None:
-        super().__init__(handshake_headers, request)
+    def __init__(self, *args, **kwargs) -> None:
```

Comment:
I prefer to be explicit about each parameter

Reply:
This is plugin implementation the explicit values are in the upper class, so I actually prefer to accept anything that is implemented on the upper class.